validate.js

```diff
@@ -315,7 +317,9 @@ class ValidationRun {
       await this.checkOscCrossRefArray(this.data, "osc:variables", "variables");
     }
     await this.checkOscCrossRefArray(this.data, "osc:missions", "eo-missions");
-    await this.checkOscCrossRefArray(this.data, "osc:themes", "themes");
+    if (typeof this.data["osc:themes"] !== 'undefined') {

```

Comment:
I'm not sure why we did this condition initially. Shouldn't osc:themes always be filled?

Reply:
If I remove it, I get an additional 6 errors:

```
  C:/Dev/open-science-catalog-metadata/projects/aeolus-innovation-aeurs-expro-lidar-measurements-to-identify-streamers-and-analyze-atmospheric-waves-lisa/collection.json
    Collection: valid
    Extensions
      osc (1.0.0-rc.3): valid
      themes (1.0.0): valid
      contacts (0.1.1): valid
    Custom
      1. 'osc:themes' must be an array
  C:/Dev/open-science-catalog-metadata/projects/aeolus-innovation-aeurs-expro-ocean-surface-wind-from-aeolus-sea-surface-returns-sea-flect/collection.json
    Collection: valid
    Extensions
      osc (1.0.0-rc.3): valid
      themes (1.0.0): valid
      contacts (0.1.1): valid
    Custom
      1. 'osc:themes' must be an array
  C:/Dev/open-science-catalog-metadata/projects/arctic-theme-3-aeur-fresh-water-fluxes/collection.json
    Collection: valid
    Extensions
      osc (1.0.0-rc.3): valid
      themes (1.0.0): valid
      contacts (0.1.1): valid
    Custom
      1. 'osc:themes' must be an array
  C:/Dev/open-science-catalog-metadata/projects/explainable-ai-application-to-trustworthy-super-resolution-opensr/collection.json
    Collection: valid
    Extensions
      osc (1.0.0-rc.3): valid
      themes (1.0.0): valid
      contacts (0.1.1): valid
    Custom
      1. 'osc:themes' must be an array
  C:/Dev/open-science-catalog-metadata/projects/sentinel-5p-innovation-aeurs-water-vapour-isotopologues-h2o-iso/collection.json
    Collection: valid
    Extensions
      osc (1.0.0-rc.3): valid
      themes (1.0.0): valid
      contacts (0.1.1): valid
    Custom
      1. 'osc:themes' must be an array
  C:/Dev/open-science-catalog-metadata/projects/stse-arctic-theme-5-aeur-contributions-to-the-year-of-polar-predictions-yopp/collection.json     
    Collection: valid
    Extensions
      osc (1.0.0-rc.3): valid
      themes (1.0.0): valid
      contacts (0.1.1): valid
    Custom
      1. 'osc:themes' must be an array
```